### PR TITLE
STCOR-883: fixed-length session timeout banner may scroll off-screen or otherwise be hidden

### PR DESCRIPTION
**Pull request by aidynoJ**
**Purpose:** 
[STCOR-883](https://folio-org.atlassian.net/browse/STCOR-883) - fixed-length session timeout banner may scroll off-screen or otherwise be hidden.
**Approach:** 
Place session timeout banner at the bottom of the page. 

Screenshot 2024-10-02 at 15 59 40
Screenshot 2024-10-02 at 16 00 05
Screenshot 2024-10-09 at 20 33 15

### CODE REVIEW

File: src/RootWithIntl.js [hunk 1]
```diff
@@ -72,10 +73,14 @@ const RootWithIntl = ({ stripes, token = '', isAuthenticated = false, disableAut
...
-                          { (typeof connectedStripes.okapi !== 'object' || connectedStripes.discovery.isFinished) && (
+                          {(typeof connectedStripes.okapi !== 'object' || connectedStripes.discovery.isFinished) && (
                             <ModuleContainer id="content">
                               <OverlayContainer />
-                              {connectedStripes.config.useSecureTokens && <SessionEventContainer history={history} queryClient={queryClient} />}
+                              {connectedStripes.config.useSecureTokens &&
+                                createPortal(<SessionEventContainer
+                                  history={history}
+                                />,
+                                document.getElementById('events-portal'))}
```
[the remark this answers is not in the document]
Comment (aidynoJ) [reply src/RootWithIntl.js]: The existing OverlayContainer is placed inside the of ModuleContainer, it is preferable have it outside of the one.

[the remark this answers is not in the document]
Comment (aidynoJ) [reply src/RootWithIntl.js]: Totally removed events-portal container and SessionEventsContainer teleports to the document.body. Based on the React community articles it is a common approach to handle modals, popups, etc

**Comment by aidynoJ** [comment]
> Can we leverage OverlayContainer instead of creating our own?
> 
> Lint is cranky.

I have changed the approach a bit. Moved createPortal to the SessionContainer itself and teleport happens into the document.body, removed events-portal and related styles. 
Lint is still cranky. I run `yarn lint --fix` before the every commit. But it doesn't fix formatting that was automatically applied by the autosaving in IDE (JB Webstorm), I'm looking into that.

File: src/RootWithIntl.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (zburke) [reply src/RootWithIntl.js]: Why?

[the remark this answers is not in the document]
Comment (aidynoJ) [reply src/RootWithIntl.js]: It is out of the main flow. I think such things should be independent from the nested content of the Root element styling etc

**Review by ryandberger — APPROVED** [review]

**Pull request merged**